readthedocs/projects/forms.py

```diff
@@ -65,8 +65,14 @@ def __init__(self, *args, **kwargs):
             coerce=lambda x: RemoteRepository.objects.get(pk=x),
             required=False,
             empty_value=None,
+            help_text=self.fields["remote_repository"].help_text,
+            label=self.fields["remote_repository"].label,
         )
 
+        # The clone URL will be set from the remote repository.
+        if self.instance.remote_repository:
+            self.fields.pop("repo")
```

Comment:
I think that @agjohnson mentioned he wanted to keep showing the URL but making it read-only.

Reply:
A read only field might be the easiest for now, yeah. We might also want to replace the remote repo field with a better widget eventually too, which would then allow us to remove the read only field later. As long as we give the maintainers are clear idea of the URL we clone from that should be fine for now though.

Reply:
The remote repository field already shows the clone URL that's being used, are we okay duplicating that information?

Reply:
Yup, that seems fine. We probably eventually just want this to be one field: when using a manual repo URL the text field shows, when using a remote repo the dropdown shows. There needs to be UI that allows projects to switch though.

So for now, showing both fields always seems the least confusing.